frontend/src/components/RequestPanel.vue

```diff
@@ -33,6 +33,8 @@ import { useListUpdate } from "@/composables/realtime"
 const activeTab = ref("My Requests")
 const socket = inject("$socket")
 
+const TAB_BUTTONS = ["My Requests", "Team Requests"] // __("My Requests"), __("Team Requests")
```

Comment:
Can you explain why you haven't translated this?

Reply:
Okay so the Tab buttons are identifiers, not labels.

Ideally we would have the following:

```js
const TAB_BUTTONS = [
  { key: "My Requests", label: __("My Requests") },
  { key: "Team Requests", label: __("Team Requests") },
]
```

but doing so is not required immediately, and would have zero impact for the end-users with minimally degraded developer experience. So I keep this for a future refactor. Perfect is the enemy of good :smiley: